### DISCUSSION TITLE
Whta does RADIO_SI4032_TX_xx_COUNT mean?

### DISCUSSION THREAD

**Discussion by pimol** [mikaelnousiainen/RS41ng#40] [Q&A] [answered]
I don't understand the meaning of TX_[MODE]_COUNT value in config.h. For example, what does "#define RADIO_SI4032_TX_CW_COUNT 1" mean?

Pietro I2OIM

**Comment by mikaelnousiainen** [comment, marked as answer]
@pimol The count settings define how many times each transmission type (modulation) is repeated.
